frontend/app/src/components/SelectChatModal.svelte

```diff
@@ -74,8 +74,10 @@
     $: selectedChatId = client.selectedChatId;
     $: globalState = client.globalStateStore;
 
+    let initialGlobalState = structuredClone($globalState);
```

Comment:
would this not work just as well without the `structuredClone`?

Reply:
Yes structuredClone was wrong